resilience4j-retry/src/main/java/io/github/resilience4j/retry/RetryConfig.java

```diff
@@ -143,10 +143,9 @@ public Builder(RetryConfig baseConfig) {
 		}
 
 		public Builder<T> waitDuration(Duration waitDuration) {
-			if (waitDuration.toMillis() < 10) {
-				throw new IllegalArgumentException("waitDurationInOpenState must be at least 10ms");
+			if (waitDuration.toMillis() >= 0) {
+				this.intervalFunction = (x) -> waitDuration.toMillis();
```

Comment:
It should throw an exception if the value is < 0

Reply:
is it possible that someone will try set value of interval with less than 0 (- value) :)?

Reply:
updated anyhow :)